AssetManagementSystem/Services/AssetService.cs

```diff
@@ -74,5 +74,11 @@ public async Task<List<Asset>> GetAllAssetsAsync()
             await _dbContext.SaveChangesAsync();
             return SelectedAsset;
         }
+
+        public async Task<List<Asset>> GetDeletedAssetsAsync()
+        {
+            var Assets = await _dbContext.Asset.ToListAsync();
```

Comment:
add filteration here since it is a business logic, always remember controller is only to get the request and pass the response.